src/core/jit/model.rs

```diff
@@ -33,12 +34,27 @@ impl<V> FromIterator<(String, V)> for Variables<V> {
 }
 
 #[derive(Debug, Clone)]
-pub struct Arg {
+pub struct Arg<Input> {
     pub id: ArgId,
     pub name: String,
     pub type_of: crate::core::blueprint::Type,
-    pub value: Option<async_graphql_value::Value>,
-    pub default_value: Option<async_graphql_value::ConstValue>,
+    pub value: Option<Input>,
```

Comment:
Why is `value` needed in the plan?

Reply:
this is the value passed from the request
![image](https://github.com/user-attachments/assets/a9587d30-9d48-4f88-8d9a-6c23c0f52dd2)

it's a part of the request and thus part of the plan